src/sardana/macroserver/macros/examples/parameters.py

```diff
@@ -135,7 +238,12 @@ def run(self, *args, **kwargs):
 class pt9(Macro):
     """Same as macro pt7 but with old style ParamRepeat. If you are writing
     a macro with variable number of parameters for the first time don't even
-    bother to look at this example since it is DEPRECATED."""
+    bother to look at this example since it is DEPRECATED.
+    Usages, ex.:
+    pt9 [[mot1 1][mot2 3]]
+    pt9 [mot1 1 mot2 3]
```

Comment:
After implementing changes agreed in the last #405 comments this should not be accepted. Please remove if you agree.